app/livechat/server/methods/getRoutingConfig.js

```diff
@@ -0,0 +1,9 @@
+import { Meteor } from 'meteor/meteor';
+
+import { RoutingManager } from '../lib/RoutingManager';
+
+Meteor.methods({
+	'livechat:getRoutingConfig'() {
```

Comment:
let's try focus on using REST endpoints instead of Meteor methods.

you don't need to change this one for merging this PR, but I'll ask to change it after it is merged ok?

Reply:
Sure, thanks.